portal-api/src/v1/helpers/escapeOperators.js

```diff
@@ -70,9 +71,15 @@ const recursivelyReplaceEscapeOperators = (filters, result = {}) => {
     } else if (typeof filters[key] === 'object' && filters[key] !== null) {
       result[key] = {};
       recursivelyReplaceEscapeOperators(filters[key], result[key]);
-      // These last two if statements handle the lowest level cases
+      // These last three if statements handle the lowest level cases
     } else if (key === 'KEYWORD') {
       result.$regex = escapeStringRegexp(filters[key]);
+    } else if (key === 'hasBeenIssued') {
+      result.hasBeenIssued = {
+        $eq: filters.hasBeenIssued,
+      };
+      // When filtering by `hasBeenIssued`, want to exclude cases where `facilityStage` is'Risk expired'
+      result.facilityStage = { $ne: FACILITY_STATUS.RISK_EXPIRED };
```

Comment:
I don't like this being necessary but I cant think of a better way to ignore `Risk expired`/`Abandoned` facilities. I think it has to be in portal-api without a big refactor of this function (to map `NE` into `$ne`) but this feels like a can of worms...

Reply:
Yeah, this is probably a larger conversation with the concept of the `hasBeenIssued` boolean... Left a comment elsewhere about it

Reply:
Oh yeah 100%, and thats an even bigger can of worms